### ISSUE TITLE
regression: ground incorrect loading from saved files

### ISSUE THREAD

**Issue by kfarr** [3DStreet/3dstreet#860]
![image](https://github.com/user-attachments/assets/836f6ebb-fb76-4a9b-9c25-7cd1381ca84a)
https://3dstreet.app/#/scenes/ea532a22-3d7d-47a1-b0ac-af99adf14abf.json

[data - 2024-09-11T113507.044.json.zip](https://github.com/user-attachments/files/16968482/data.-.2024-09-11T113507.044.json.zip)

**Comment by kfarr** [comment]
I had to revert this change:
https://github.com/3DStreet/3dstreet/commit/507fdb91f4d61f280cea59698779ed7112688111

The impact was that the component values for position were not being stored when saving the files, resulting in reloaded scenes with values of 0 0 0 for entities such as ground grass, ground asphalt, etc. It also affected pedestrian height placement but this is not as noticeable to end-users. This issue only appeared after a user ends a session and reloads the scene in a new session.

**Comment by vincentfretin** [comment]
Oh wow okay, it makes sense, there was no position component anymore so it was not saved.
We should really have an automated non regression test case for this stuff. I wrote some tests with [vitest](https://vitest.dev/) in my project with `expect(createdJson).toMatchInlineSnapshot(expectedJson)` ([doc](https://vitest.dev/guide/snapshot.html)) for functions that didn't need browser DOM api, that helped a lot. For running test in browser, vitest can be run in chromium browser with playwright, I just tested a simple react test, that seems to work great https://vitest.dev/guide/browser/
That's a modern test tools alternative to nyc, mocha, chai, approvals, karma, jsdom, jest.
